test-suite/js/src/test/scala/org/scalajs/testsuite/library/JsBigIntTest.scala

```diff
@@ -0,0 +1,137 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Test
+import org.scalajs.testsuite.utils.JSAssert.assertJSUndefined
+
+import scala.scalajs.js
+
+class JsBigIntTest {
+
+  // Methods we actually implement
+
+  @Test def apply(): Unit = {
+    val bi = js.BigInt("9007199254740992")
+    assertEquals(bi.toString(), "90071992547409912")
+  }
+
+  @Test def asIntN(): Unit = {
+    val x = js.BigInt(2) ** js.BigInt(64)
+    assertEquals(js.BigInt.asIntN(256, x), js.BigInt("18446744073709551616"))
+  }
+
+  @Test def asUintN(): Unit = {
+    val x = js.BigInt(123)
+    assertEquals(js.BigInt.asUintN(50, x), js.BigInt("1125899906842501"))
+  }
+
+  @Test def toLocaleString(): Unit = {
+    val bi = js.BigInt("42123456789123456789")
+    assertEquals(bi.toString(), "90071992547409912")
+
+    val currency = js.BigInt("42123456789123456789")
+      .toLocaleString("de-DE", js.BigIntOptions(
+        style = "currency",
+        currency = "EUR"
+      ))
+    // Options may not be supported
+    assertTrue(currency == "42123456789123456789"
+        || currency == "42.123.456.789.123.456.789,00 €")
+  }
+
+  @Test def valueOf(): Unit = {
+    val bi = js.BigInt("42123456789123456789")
+    assertEquals(bi, bi.valueOf())
+  }
+
+  @Test def operators(): Unit = {
+    val previousMaxSafe = js.BigInt("9007199254740991");
```

Comment:
You have a bunch of stray `;` here ;)

Reply:
Removed in https://github.com/scala-js/scala-js/pull/3786/commits/2cd388f224fff3c4aac0e44f1bd8f2373b314c69